docs/site/tutorials/connectors/Mongodb-tutorial.md

```diff
@@ -0,0 +1,281 @@
+? Application class name: MyAppApplication
+? Select features to enable in the project (Press <space> to select, <a> to togg
+le all, <i> to invert selection)
+❯◉ Enable eslint: add a linter with pre-configured lint rules
+ ◉ Enable prettier: install prettier to format code conforming to rules
+ ◉ Enable mocha: install mocha to run tests
+ ◉ Enable loopbackBuild: use @loopback/build helpers (e.g. lb-eslint)
+ ◉ Enable vscode: add VSCode config files
+ ◉ Enable docker: include Dockerfile and .dockerignore
+ ◉ Enable repositories: include repository imports and RepositoryMixin
+(Move up and down to reveal more choices)
+```
+
+### 2. Create models
+
+Let's create a simple model `User`. To keep the tutorial short, the prompts of
+`lb4 model` are skipped:
+
+```ts
+// imports
+@model()
+export class User extends Entity {
+  @property({
+    type: 'number',
+    id: true,
+    generated: true,
+  })
+  id?: number;
+
+  @property({
+    type: 'string',
+  })
+  name?: string;
+
+  @property({
+    type: 'boolean',
+    required: true,
+  })
+  hasAccount: boolean;
+
+  constructor(data?: Partial<User>) {
+    super(data);
+  }
+}
+```
+
+### 3. Create a data source
+
+Next, let's create a dataSource `db` using MongoDB connector with the following
+setups:
+
+```bash
+$ lb4 datasource
+? Datasource name: db
+? Select the connector for db:
+  ...
+  Redis key-value connector (supported by StrongLoop)
+❯ MongoDB (supported by StrongLoop)
+  MySQL (supported by StrongLoop)
+  ...
+? Connection String url to override other settings (eg: mongodb://username:passw
+ord@hostname:port/database):
+? host: localhost
+? port: 27017
+? user:
+? password: [hidden]
+? database: demo
+? Feature supported by MongoDB v3.1.0 and above: Yes
+
+Datasource Db was created in src/datasources/
+```
+
+Under `src/datasources/db.datasource.ts`, we can find the `DBDataSource` class
+and the config we just set:
+
+```ts
+const config = {
+  name: 'db',
+  connector: 'mongodb',
+  url: '',
+  host: 'localhost',
+  port: 27017,
+  user: '',
+  password: '',
+  database: 'demo',
+  useNewUrlParser: true,
+};
+```
+
+{% include important.html content="please make sure you are using `loopback-connector-mongodb` package version 5.2.1
+or above to handle `ObjectId` properly." %}
+
+### 4. Create repositories
+
+[Repository](../../Repository.md) is a artifact that ties the model and the
+datasource. We will need to create the repository for the `User` class before
+accessing the database. The steps of creating `UserRepository` by running
+`lb4 repository` are skipped here:
+
+```ts
+// imports
+export class UserRepository extends DefaultCrudRepository<
+  User,
+  typeof User.prototype.id,
+  UserRelations
+> {
+  constructor(@inject('datasources.db') dataSource: DbDataSource) {
+    super(User, dataSource);
+  }
+}
+```
+
+<!-- From what I tested, mongoDB doesn't need migration, and `npm run migrate` has no effect on the existing collections as well. -->
```

Comment:
Will remove this part once it is confirmed.

Reply:
no migration in MongoDB connector